package.json

```diff
@@ -52,8 +52,14 @@
     ]
   },
   "dependencies": {
+    "@aws-sdk/client-acm": "^3.667.0",
+    "@aws-sdk/client-cloudfront": "^3.667.0",
+    "@aws-sdk/client-iam": "^3.667.0",
+    "@aws-sdk/client-lambda": "^3.667.0",
+    "@aws-sdk/client-route-53": "^3.667.0",
+    "@aws-sdk/client-s3": "^3.667.0",
+    "@aws-sdk/smithy-client": "^3.374.0",
```

Comment:
I wanted to know what was smithy and I found that it was deprecated: https://www.npmjs.com/package/@aws-sdk/smithy-client replaced by: 
![image](https://github.com/user-attachments/assets/523ed5ba-acfa-4e6c-b09a-6ad855c916ad)
That is not suppose to be used 😬

Reply:
removed ! 👍🏻